index.html

```diff
@@ -33,5 +40,124 @@ <h2>LET'S CONNECT</h2>
+            <li class="html">HTML</li>
+            <li class="css">CSS</li>
+            <li class="js">JavaScript</li>
+          </ul>
+          <button class="seepro">See project</button>
+        </div>
+        <div class="grid_item">
+          <img src="/Asset/avail.svg" alt="" />
+          <h2>Multi-Post stories</h2>
+          <ul class="can">
+            <li class="canopy">CANOPY</li>
+            <li class="backend">Back End Dev</li>
+            <li class="twenty">2015</li>
+          </ul>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <ul class="lang">
+            <li class="html">HTML</li>
+            <li class="css">CSS</li>
+            <li class="js">JavaScript</li>
+          </ul>
+          <button class="seepro">See project</button>
+        </div>
+        <div class="grid_item">
+          <img src="/Asset/oval.svg" alt="" />
+          <h2>Tonic</h2>
+          <ul class="can">
+            <li class="canopy">CANOPY</li>
+            <li class="backend">Back End Dev</li>
+            <li class="twenty">2015</li>
+          </ul>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <ul class="lang">
+            <li class="html">HTML</li>
+            <li class="css">CSS</li>
+            <li class="js">JavaScript</li>
+          </ul>
+          <button class="seepro">See project</button>
+        </div>
+        <div class="grid_item">
+          <img src="/Asset/prof.svg" alt="" />
+          <h2>Multi-Post Stories</h2>
+          <ul class="can">
+            <li class="canopy">CANOPY</li>
+            <li class="backend">Back End Dev</li>
+            <li class="twenty">2015</li>
+          </ul>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <ul class="lang">
+            <li class="html">HTML</li>
+            <li class="css">CSS</li>
+            <li class="js">JavaScript</li>
+          </ul>
+          <button class="seepro">See project</button>
+        </div>
+      </div>
+    </section>
+    <section class="about">
+      <div class="about_me">
+        <h2>About MySelf</h2>
+        <p>
+          Hello I’m a software developer! I can help you build a product ,
+          feature or website. Look through some of my work and experience! If
+          you like what you see and have a project you need coded, don’t
+          hesitate to contact me.
+        </p>
+        <h3>LET'S CONNECT</h3>
+      </div>
+      <div class="socialm">
+        <img src="/Asset/github.svg" alt="" />
+        <img src="/Asset/linkedin.svg" alt="" />
+        <img src="/Asset/angellist.svg" alt="" />
+        <img src="/Asset/twitter.svg" alt="" />
+        <img src="/Asset/meduim.svg" alt="" />
+      </div>
```

Comment:
- [ ] I think it would be better to use `<ul>` and `<li>` tags to group and list the social icons in your `about me` . This is important for Semantic structure. The `<ul>` (unordered list) and `<li>` (list item) tags are part of the semantic structure of HTML, which helps to make your content more organized and easier to understand for both humans and machines. By using these tags, you are indicating to the browser and other devices that the content is a list of items, which improves accessibility and search engine optimization.
- [ ] You can then reduce the size of the icons as they appear to be bigger compared to the original design

![image](https://user-images.githubusercontent.com/98453979/227637041-6f4d6176-bc5e-418e-882b-7f87312249b2.png)


### Checkout the original design
![image](https://user-images.githubusercontent.com/98453979/227637128-a14c6005-1ff4-4779-ac92-ec5d0c6ade7f.png)